kythe/cxx/indexer/textproto/analyzer.h

```diff
@@ -55,6 +55,17 @@ absl::Status AnalyzeCompilationUnit(const proto::CompilationUnit& unit,
                                     const std::vector<proto::FileData>& files,
                                     KytheGraphRecorder* recorder);
 
+// Callback function to instantiate plugins for a given proto message type.
+using PluginLoadCallback = std::function<std::vector<std::unique_ptr<Plugin>>(
+    std::string msg_name, const google::protobuf::Message& proto)>;
```

Comment:
`absl::string_view`?

Reply:
Also, if the function itself doesn't have to outlive the call here, I might consider using https://github.com/abseil/abseil-cpp/blob/master/absl/functional/function_ref.h instead.  It has better semantics than std::function, but it's pretty minor.

Reply:
Done.